cross-chain/arbitrum/deploy_l2/13_transfer_proxy_admin_ownership.ts

```diff
@@ -0,0 +1,25 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { helpers, upgrades, deployments } = hre
+  const { governance, deployer } = await helpers.signers.getNamedSigners()
+  const { log } = deployments
+
+  const proxyAdmin = await upgrades.admin.getInstance()
+  const currentOwner = await proxyAdmin.owner()
+
+  // The `@openzeppelin/hardhat-upgrades` plugin deploys a single ProxyAdmin
+  // per network. We don't want to transfer the ownership if the owner is already
+  // set to the desired address.
+  if (!helpers.address.equal(currentOwner, governance.address)) {
+    log(`transferring ownership of ProxyAdmin to ${governance.address}`)
+    await (
+      await proxyAdmin.connect(deployer).transferOwnership(governance.address)
```

Comment:
In the core `solidity` pkg, we transfer the ProxyAdmin ownership to the ESDM. Don't we want to do the same here?

Reply:
Nope, I asked and we don't need `esdm`. https://discord.com/channels/893441201628405760/1078358605524369489/1079716885735678003

Reply:
ESDM was never deployed, let's transfer the ownership to Threshould Council, see https://github.com/keep-network/tbtc-v2/issues/549.